htlc/src/components/withdrawHtlc.js

```diff
@@ -0,0 +1,62 @@
+/* global AlgoSigner */
+import { LEDGER } from '../algosigner.config';
+import * as YAMLData from '../../artifacts/cache/htlc.py.yaml';
+
+const algosdk = require("algosdk");
+const token = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
+const server = "127.0.0.1";
+const port = 4001;
```

Comment:
We should import these from `algob.config.js`

Reply:
Okay